static/src/stylesheets/module/_discussion.scss

```diff
@@ -903,6 +917,54 @@ $comment-recommend-button-size: 19px;
     }
 }
 
+.sharing-component {
+    height: 1.2rem;
+    vertical-align: middle;
+    margin-left: 5px;
+
+    .sharing-buttons {
+        opacity: 0;
+        cursor: pointer;
+        transition: opacity 0s ease-in-out;
+        display: inline-block;
+
+        .inline-icon {
+            height: 0rem;
+            min-width: 0rem;
+            transition: all .3s ease-in-out;
+        }
+    }
+
+    .sharing-text, .inline-icon {
+        cursor: default;
+        display: inline-block;
+        float: left;
+    }
+
+    .sharing-text {
+        .inline-icon {
+            padding: 1px 5px;
+            fill: #005689;
+        }
+    }
+
+    a:first-child {
+        margin-left: 5px;
+    }
+}
+
+.sharing-component:hover {
```

Comment:
It's best to try and keep the 'state' styles with your component - so you can nest `&:hover` inside your sharing component block above